Include/Acidanthera/Library/OcVirtualFsLib.h

```diff
@@ -66,6 +94,82 @@ CreateRealFile (
   IN OUT EFI_FILE_PROTOCOL  **File
   );
 
+/**
+  Creates read-only EFI_FILE_PROTOCOL virtual directory instance,
+  optionally as an overlay over an existing EFI_FILE_PROTOCOL instance.
```

Comment:
I do not see `optionally as an overlay over an existing EFI_FILE_PROTOCOL instance` in the code. `File` is always overwritten regardless of the passed value. What do I miss?